provider/resources.go

```diff
@@ -143,7 +151,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 	}
 
-	err := prov.ComputeDefaults(tfbridge.TokensSingleModule("ns1_", mainMod, func(module, name string) (string, error) {
+	err := x.ComputeDefaults(&prov, x.TokensSingleModule("ns1_", mainMod, func(module, name string) (string, error) {
 		return tfbridge.MakeResource(mainPkg, module, name).String(), nil
 	}))
```

Comment:
While we're here, we can put in the standard token mapper:

```suggestion
	err := x.ComputeDefaults(&prov, x.TokensSingleModule("ns1_", mainMod, s.MakeStandardToken(mainPkg)))
```

It does the same thing as what we have now, but it looks cleaner.